### ISSUE TITLE
`error()` is broken after running `exec()`

### ISSUE THREAD

**Issue by amjibaly** [shelljs/shelljs#156]
Something seems broken with the `error()` function after running exec. I would expect `error()` to always return something if `exec().code !== 0` but that is not the case. Some sample output from the console:

```js
> x = sh.exec('curl -s -S capoie')
curl: (6) Could not resolve host: capoie
{ code: 6,
  output: 'curl: (6) Could not resolve host: capoie\n' }
> sh.error()
null
```

It works better with a command like `cd` but the output from `error()` is still incomplete:
```js
> x = sh.exec('cd sfs')
/bin/sh: line 0: cd: sfs: No such file or directory
{ code: 1,
  output: '/bin/sh: line 0: cd: sfs: No such file or directory\n' }
> sh.error()
'exec: \n'
> x.output
'/bin/sh: line 0: cd: sfs: No such file or directory\n'
```

**Comment by ariporad** [comment]
@amjibaly: That's because `error()` just returns `stderr`. This will be fixed by #269.

**Comment by ariporad** [comment]
I'm going to close this, as it will be fixed.
